install/ui/src/freeipa/serverconfig.js

```diff
@@ -49,6 +49,23 @@ return {
                         'ipasearchtimelimit'
                     ]
                 },
+                {
+                    name: 'server',
+                    label: '@i18n:objects.config.server',
+                    fields: [
+                        {
+                            $type: 'entity_select',
+                            name: 'ca_renewal_master_server',
+                            other_entity: 'server',
+                            other_field: 'cn',
+                            flags: ['w_if_no_aci']
+                        },
+                        {
+                            $type: 'list',
```

Comment:
You could also use
```
$type: 'multivalued'
name: 'pkinit_server_server',
read_only: true
```
instead of defining a new `list` widget. For instance this is used to display the IPA DNS servers in the DNS Global Configuration page. Why did you make this choice, is `multivalued` missing a functionality?

Reply:
Yes, you're right. Seems 'multivalued' is fit. I'll update PR.

Reply:
Fixed